include/oxen/quic/endpoint.hpp

```diff
@@ -88,7 +88,7 @@ namespace oxen::quic
             std::promise<std::shared_ptr<Connection>> p;
             auto f = p.get_future();
 
-            if (!remote.is_addressable())
+            if (not _manual_routing and !remote.is_addressable())
```

Comment:
```suggestion
            if (!_manual_routing && !remote.is_addressable())
```
The mixed keyword and non-keyword operators are jarring (and currently everything else in libquic uses symbols instead of keywords).

Side note: I looked for a way to get clang-format to enforce this, but couldn't find one.